### DISCUSSION TITLE
Change in specified inlet boundary condition value

### DISCUSSION THREAD

**Discussion by MeghaBha** [su2code/SU2#1744] [Q&A]
Hello everyone,

I am running a supersonic , viscous flow simulation over a supersonic intake using SU2 v7.3 for which I have specified the inlet boundary condition as MARKER_SUPERSONIC_INLET. After the simulation, I noticed that the values at the inlet are not the ones that I specified. For instance, if specified Mach number at inlet is 2.5, simulation results show it as 2.48. If specified static pressure is 21691.12 Pa, results display a value of 22031.8 Pa. I am facing this issue no matter how refined the grid is. Could anyone tell me the reason for this change and how to rectify it?

Thanks

**Comment by pcarruscag** [comment]
Hi,
Inlets are weak boundary conditions (imposed as a flux) therefore the solution values at inlet nodes depend on grid refinement and also on how far the boundary is from significant flow features (further away means closer to specified values).

**Reply by MeghaBha** [reply]
Thanks for your reply, @pcarruscag. I also ran the "Inviscid supersonic wedge" tutorial with 21692.142 Pa, 216.2 K and 2.5 Mach conditions in SU2 for both viscous and inviscid flows. I noticed that the change in specified values at inlet is more pronounced for viscous flow simulation than for inviscid as shown in the attached images. The images depict the values at a mesh node located near the inlet. What could be the reason for this anomaly?
INVISCID CASE:
![MicrosoftTeams-image (12)](https://user-images.githubusercontent.com/90450995/187071822-6689ad3a-50ca-4f9e-9db9-b96a3868d53e.png)
VISCOUS CASE
![MicrosoftTeams-image (13)](https://user-images.githubusercontent.com/90450995/187071830-a3f1045a-7da4-4fca-960e-10ab5e6253ae.png)

**Reply by pcarruscag** [reply]
A viscous wall counts as a significant flow feature, the results are better if you extend the inlet a bit and place a small slip wall before the viscous wall, as done here
https://su2code.github.io/vandv/swbli/
